### ISSUE TITLE
Zooming using zoomTo is "laggy"

### ISSUE THREAD

**Issue by abithyzis** [jaukia/zoomooz#64]
I have a grid of elements to which I zoom using the `zoomTo` function. I notice that zooming in is not smooth.

Looking at this http://www.parti-socialiste.fr/bilan-sarkozy/ (which is similar to what I try to achieve) the only difference I see is that they have a root different than `$(document.body)`. They set the root like so

``` javascript
var $zoomoozContainer = $('#rickrollzarkozing')
$.zoomMooz.setup({
    root: $zoomoozContainer,
    targetsize: 0.8
});
```

When I try to change that root in a similar fashion I get the following javascript error

> NS_ERROR_XPC_BAD_CONVERT_JS: Could not convert JavaScript argument arg 0 [nsIDOMWindow.getComputedStyle]

As per documentation I have set `position:relative` to the element I want as root.

So basically my q is two-fold.
1. Will changing the root from body make any difference in terms of smoothness
2. Why changing the root fails in my case?

I can provide you with a url to see this behavior but I can only do it in private.
